CMakeLists.txt

```diff
@@ -82,13 +82,6 @@ find_library(
   HINTS "${CMAKE_PREFIX_PATH}/lib" )
 message("amdhip64: ${LIBAMDHIP64_LIBRARY}")
```

Comment:
Do we need this?

Reply:
yes, we do need it to cover the case where libamdhip64.so doesn't exist on the user's stack, or if they're using a different runtime environment